src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -408,8 +411,9 @@ void ICACHE_RAM_ATTR SX1280Driver::TXnb()
 #endif
 }
 
-void ICACHE_RAM_ATTR SX1280Driver::RXnbISR()
+void ICACHE_RAM_ATTR SX1280Driver::RXnbISR(uint16_t const irqStatus)
 {
+    uint8_t const fail = !!(irqStatus & (SX1280_IRQ_CRC_ERROR | SX1280_IRQ_RX_TX_TIMEOUT));
```

Comment:
Should the type be `bool`? And on the callback function parameters too.

Reply:
You are right. Type could be the `bool` atm but it does not save anything and `uint8_t` can be used to do the same job.
I left an option to use it as a flag later if needed and that is the reason for `uint8_t`.

Reply:
Thats fine, maybe need to introduce some constants then so it is known now.

Reply:
hmm, not a bad idea.
I will modify it and need to introduce new common radio header to keep that aligned...